meta_merkle_tree/src/generated_merkle_tree.rs

```diff
@@ -43,31 +51,34 @@ pub struct GeneratedMerkleTree {
 }
 
 impl GeneratedMerkleTreeCollection {
-    pub fn new_from_stake_meta_collection(
+    pub async fn new_from_stake_meta_collection(
```

Comment:
does this need to be async?

Reply:
This is marked as resolved, but async should still be deleted. The `meta_merkle_tree` crate will not use EllipsisClient